api/app/src/providers/oauth2.ts

```diff
@@ -16,6 +16,7 @@ export const oauthUserTokenResponse = z.object({
 
 export interface OAuth2 {
   getAuthUri(state: string): Promise<string>;
+  revokeProviderAccess(connectedUser: ConnectedUser): Promise<void | string>;
```

Comment:
Curious, why returning `void | string`? It seems we don't need to return `string` (maybe just add a `await` on Withings':

```
    if (status === 0) {
      await this.oauth.revokeLocal(connectedUser);
      return;
    }
```

Ideally we wouldn't adjust the interface's contract bc of a specific implementation.